10/graph-components.checkpoint.md

```diff
@@ -0,0 +1,333 @@
+# Implement Components of a Graph
+
+## Problem
+
+Given an undirected graph represented by an adjacency dictionary, `g`, return a list of lists containing all of the subgraphs in the graph. Each element in the outer list will represent a subgraph in the graph. Each element in the inner lists will contain all of the nodes in a subgraph. Please note the items in the inner lists must be sorted.
```

Comment:
I might also suggest specifying that the outer resulting list doesn't need to be sorted. 
```suggestion
Given an undirected graph represented by an adjacency dictionary, `g`, return a list of lists containing all of the subgraphs in the graph. Each element in the outer list will represent a subgraph in the graph. Each element in the inner lists will contain all of the nodes in a subgraph. Please note the items in the inner lists must be sorted from least to greatest.
```